dev-docs/examples/postbid.md

```diff
@@ -9,8 +9,8 @@ sidebarType: 1
 why_link: /overview/what-is-post-bid.html
 
 about: 
-- Postbid is a third-party tag creative you setup in your ad server. Create one line item in the ad server targeting each ad unit on your page. This line item's <strong>creative will contain the code below</strong>.
-- If your ad unit supports multiple sizes, create multiple Postbid creatives for each size. 
+- Postbid is a third-party tag creative you setup in your ad server that loads the whole Prebid.js package. Create a line item in the ad server targeting each ad unit on your page. The <strong>creative will contain the code below</strong>.
+- If the ad unit supports multiple sizes, you'll need to do one of two things -- either create a Postbid creative for each desired size with that size hardcoded in the PBJS adunit, or use ad server macros to pass the size through to the creative. e.g. in GAM, there are HEIGHT and WIDTH macros. 
```

Comment:
could we add a reminder not to use the  not-for-prod js in the fiddle here as well?

Reply:
done. there are a dozen other fiddles. will add that to the pile for the contract tech writer we're getting next month